### ISSUE TITLE
Default file system changes in read_msd

### ISSUE THREAD

**Issue by ifellows** [ICPI/ICPIutilities#53]
`read_msd` by default creates a new `.rds` file. Now it also deletes the existing `.txt` file from the file system.

As a matter of practice, I don't think a function should mess around with the file system as an unrelated side effect to its stated purpose. It especially shouldn't delete any files. The result is that If you have code that reads in data and then performs an analysis, the code will fail on the second run because the data it was expecting had been deleted. Also, the renaming scheme to create these files is a bit fragile. For example:

```
> ICPIutilities:::rename_msd("renamed_mer_txt_data.txt")
[1] "renamed_mer_rds_data.txt"
```

**Comment by achafetz** [comment]
Thanks, and sorry I missed this one during the chaos of COP.  

From the typical use case, we usually read in the MSD when its rolled out and convert to rds to save on space and remove the text file. So, this makes sense for most of us 90% of the time.

I completely see it from your side. Seems like a silly idea from a reproducible analysis stand point to have the default be to delete the raw file. 

I hear you and will change the defaults later this week. Thanks

**Comment by achafetz** [comment]
resolved by #55
